### PR TITLE
fix oled for qt py with secondary i2c

### PR DESCRIPTION
**Pull request by ladyada**

### CODE REVIEW

**Comment by jedgarpark** [comment]
all good now

**Review by jedgarpark — COMMENTED** [review]
The issue with display not showing up was fixed w the TFT on by default change to setup.
Couldn't get filesystem to show up so added files, edited json via CircuitPython install.

**Comment by jedgarpark** [comment]
seems like this is waiting for serial monitor to open before starting the server/printing to display

File: Adafruit_ESP32S2_TFT_WebServer/Adafruit_ESP32S2_TFT_WebServer.ino [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+
+#define DEFAULT_AP_PASSWORD "MY_AP_PASSWORD"
+char ap_password[80] = DEFAULT_PASSWORD;
+
+#define DEFAULT_HOSTNAME "esp32sd"
+char hostname[80] = DEFAULT_HOSTNAME;
+
+StaticJsonDocument<512> doc;
+
+#define EXPOSE_FS_ON_MSD
```
Comment (caternuson) [inline Adafruit_ESP32S2_TFT_WebServer/Adafruit_ESP32S2_TFT_WebServer.ino:54]: I had to comment this out to prevent a WDT reset loop.

**Review by caternuson — APPROVED** [review]

**Pull request merged**